src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -206,17 +208,18 @@ export const amountQuoteFiatPlugin: FiatPluginFactory = async (params: FiatPlugi
             goodQuotes.push(quote)
           }
 
+          const noQuoteText = direction === 'buy' ? lstrings.fiat_plugin_buy_no_quote : lstrings.fiat_plugin_sell_no_quote
```

Comment:
Move error string logic into getBestError/getErrorText

Reply:
This would duplicate the logic in the two funcs getBestError/getErrorText. Where it is now keeps the logic consistent.